### PR TITLE
CNV-5774: Add links to tuning and scaling guide

### PR DESCRIPTION
**Pull request by sabrinajess**
Version(s): 4.13+

Issue: [CNV-5774](https://issues.redhat.com//browse/CNV-5774)

Link to docs preview:
https://59567--docspreview.netlify.app/openshift-enterprise/latest/virt/about-virt.html
https://59567--docspreview.netlify.app/openshift-enterprise/latest/virt/virt-getting-started.html
https://59567--docspreview.netlify.app/openshift-enterprise/latest/virt/live_migration/virt-live-migration.html

QE review:
- [] QE has approved this change. No QE required - just adding links

Additional information:

### CODE REVIEW

File: virt/virt-getting-started.adoc [hunk 1]
```diff
@@ -70,3 +70,4 @@ VMs are connected to the pod network by default. You must configure a secondary
 * xref:../virt/virtual_machines/virt-automating-windows-sysprep.adoc#virt-automating-windows-sysprep[Automate Windows VM deployments with `sysprep`].
 * xref:../virt/live_migration/virt-live-migration.adoc#virt-live-migration[Live migrate VMs].
 * xref:../virt/backup_restore/virt-backup-restore-overview.adoc#virt-backup-restore-overview[Back up and restore VMs].
+* link:https://access.redhat.com/articles/6994974[Tune and scale your cluster]
```
Comment (stevsmit) [inline virt/virt-getting-started.adoc:73]: This link and the other lead to the same page, but the friendly reference text is different. Just wondering if you think it should be the same, or as is.

**Comment by stevsmit** [comment]
@sabrinajess 1 comment. Otherwise, lgtm

File: virt/virt-getting-started.adoc (hunk 1 above)
Comment (sabrinajess) [reply virt/virt-getting-started.adoc:73]: I changed the link text on this one from the title of article to match the style of the other links in the resources section so it didn't stick out like a lil odd duck.

**Review by maxwelldb — APPROVED** [review]

**Comment by maxwelldb** [comment]
@sabrinajess You'll need to manually cherrypick this onto enterprise-4.13.

**Pull request merged**